README.md

```diff
@@ -54,7 +62,61 @@ The extension works with 'list of text' properties and supports the following pa
     }
   }
 }
+```
+
+### Hybris
+
+Hybris works with a list of objects with the properties id and catalog and requires the following parameters:
+
+```json
+{
+  "hybrisUrl": "{URL of your hybris api}",
+  "basePath": "{base path of the version of your hyrbis url i.e /rest/v2}",
```

Comment:
don't think we should be exposing this as a configuration param, the extension has been designed to work with v2 of the API so we should just hardcode that in the extension. If they want to change it, they can fork the repo.